src/pages/iou/IOUDetailsModal.js

```diff
@@ -99,14 +99,27 @@ class IOUDetailsModal extends Component {
      * @param {String} paymentMethodType
      */
     performIOUPayment(paymentMethodType) {
-        IOU.payIOUReport({
-            chatReportID: this.props.route.params.chatReportID,
-            reportID: this.props.route.params.iouReportID,
-            paymentMethodType,
-            amount: this.props.iouReport.total,
-            currency: this.props.iouReport.currency,
-            requestorPayPalMeAddress: this.props.iouReport.submitterPayPalMeAddress,
-        });
+        const recipient = {
+            login: this.props.iouReport.ownerEmail,
+            payPalMeAddress: this.props.iouReport.submitterPayPalMeAddress,
```

Comment:
Should we use some null coallesence here in case the other user does not have paypal address

Reply:
The PayPal.me button will not show up if the other user does not have a PayPal address added to their account.